### PR TITLE
Disabling tests/run-make/compiler-builtins for sgx target(x86_64-fortanix-unknown-sgx)

### PR DESCRIPTION
**Pull request by NirjharRoyiitk**
Disabling tests/run-make/compiler-builtins for sgx target(x86_64-fortanix-unknown-sgx).

### CODE REVIEW

**Comment by NirjharRoyiitk** [comment]
cc: @raoulstrackx @jethrogb

File: tests/run-make/compiler-builtins/rmake.rs [hunk 1]
```diff
@@ -38,6 +38,14 @@ fn main() {
     let target_dir = tmp_dir().join("target");
     let target = std::env::var("TARGET").unwrap();
 
+    if target.starts_with("x86_64-fortanix-unknown-sgx") {
```
Comment (jieyouxu) [inline tests/run-make/compiler-builtins/rmake.rs:41]: I believe you can use a compiletest directive in rmake.rs:

//@ ignore-sgx

(please leave either a reason/fixme too)

Comment (NirjharRoyiitk) [reply tests/run-make/compiler-builtins/rmake.rs:46]: done

Comment (jieyouxu) [reply tests/run-make/compiler-builtins/rmake.rs:46]: (you shouldn't need the target check and return in main anymore)

Comment (NirjharRoyiitk) [reply tests/run-make/compiler-builtins/rmake.rs]: Yeah sorry. I forgot to remove it in the previous push. I have made the changes. Does it look fine now? @jieyouxu

Comment (jieyouxu) [reply tests/run-make/compiler-builtins/rmake.rs]: Yeah LGTM now

**Comment by Dylan-DPC** [comment]
@NirjharRoyiitk any updates on this? thanks

**Comment by NirjharRoyiitk** [comment]
not required

**Pull request closed without merging**